src/QmlControls/CommonHiddenTopBar.qml

```diff
@@ -0,0 +1,163 @@
+        anchors.top:            parent.top
+        anchors.bottom:         parent.bottom
+        width: parent.width
+        anchors.fill: parent
+
+        Button {
+            width: viewButtonRow.width
+            height: parent.height
+            background: Rectangle {
+                color: "transparent"
+                Row {
+                    Rectangle {
+                        height: parent.height - 20
+                        anchors.verticalCenter: parent.verticalCenter
+                        Row {
+                            anchors.fill: parent
+                            anchors.leftMargin: viewButtonRow.width / 6 - 80
+                            spacing: 10
+                            Image {
+                                source: "/res/TopMenu_gps_status.svg"
+                                height: 40
+                                width: height
+                            }
+                            Image {
+                                source: "/res/TopMenu_signalLevel.svg"
+                                height: 40
+                                width: height
+                            }
+                            anchors.bottom: parent.bottom
+                        }
+                    }
+
+                    Rectangle {
+                        height: parent.height - 20
+                        anchors.verticalCenter: parent.verticalCenter
+                        Row {
+                            anchors.fill: parent
+                            anchors.leftMargin: viewButtonRow.width / 6 * 2 - 80
+                            spacing: 10
+                            Image {
+                                source: "/res/TopMenu_Remote_controller_connection_status.svg"
+                                height: 40
+                                width: height
+                            }
+                            Image {
+                                source: "/res/TopMenu_signalLevel.svg"
+                                height: 40
+                                width: height
+                            }
+                            anchors.bottom: parent.bottom
+                        }
+                    }
+                    Rectangle {
+                        height: parent.height - 20
+                        anchors.verticalCenter: parent.verticalCenter
+                        Row {
+                            anchors.fill: parent
+                            anchors.leftMargin: viewButtonRow.width / 6 * 3 - 80
+                            spacing: 10
+                            Image {
+                                source: "/res/TopMenu_Image_transfer_status.svg"
+                                height: 40
+                                width: height
+                            }
+                            Image {
+                                source: "/res/TopMenu_signalLevel.svg"
+                                height: 40
```

Comment:
Don't hardcode the height => Use property binding of ScreenTool constants. Check all the following